insights/parsers/tests/test_bond.py

```diff
@@ -168,3 +170,5 @@ def test_bond_class():
     bond_obj_2 = Bond(context_wrap(BONDINFO_MODE_6, CONTEXT_PATH))
     assert bond_obj_2.bond_mode == '1'
     assert bond_obj_2.active_slave is None
+
+    bond_obj_2 = Bond(context_wrap(BONDINFO_UNKNOWN_BOND_MODE, CONTEXT_PATH))
```

Comment:
Better add a check for the `unknown` bond mode

Reply:
@xiangce - Done